airbyte-integrations/connectors/source-s3/metadata.yaml

```diff
@@ -23,7 +23,7 @@ data:
       packageName: airbyte-source-s3
   registries:
     cloud:
-      dockerImageTag: 4.4.1
+      dockerImageTag: 4.5.2
```

Comment:
We should remove this line completely, otherwise it'll stay pinned for cloud to this new version

Reply:
Whoops, thank you! Fixed.